apps/modernization-ui/src/apps/patient/data/address/AddressEntryFields.spec.tsx

```diff
@@ -70,67 +68,4 @@ describe('AddressEntryFields', () => {
         expect(getByLabelText('Country')).toBeInTheDocument();
         expect(getByLabelText('Address comments')).toBeInTheDocument();
     });
-
-    it('should require type', async () => {
```

Comment:
issue:  These tests are still useful, the fields are still required they are just validated on submit rather than blur